products/spanner/api.yaml

```diff
@@ -12,8 +12,8 @@
 # limitations under the License.
 
 --- !ruby/object:Api::Product
-name: Google Spanner
-prefix: gspanner
+name: Spanner
+display_name: Cloud Spanner
```

Comment:
How did you pick Cloud Spanner vs Google Cloud SQL?  I have no preference there.

Reply:
Google Spanner was obviously wrong, so I checked the name & it was Cloud Spanner. Google Cloud SQL seemed like a real product name so I preserved that but it's also wrong - it's just Cloud SQL. Filed https://github.com/GoogleCloudPlatform/magic-modules/issues/1209 to make a pass and fix these all independently of code changes.